### PR TITLE
Remove version restrictions of sagemaker-containers

### PR DESCRIPTION
**Pull request by yangaws**
*Issue #, if available:*

*Description of changes:*
Remove the version restrictions of sagemaker-containers in setup.py

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def read(fname):
         'Programming Language :: Python :: 3.5',
     ],
 
-    install_requires=['numpy', 'sagemaker-containers==2.1.0', 'Pillow', 'retrying', 'six'],
+    install_requires=['numpy', 'sagemaker-containers', 'Pillow', 'retrying', 'six'],
```
Comment (laurenyu) [inline setup.py]: should we put a minimum?

Comment (nadiaya) [reply setup.py]: +1

Comment (yangaws) [reply setup.py]: Talked offline. Set lower bound to 2.1.0

**Comment by laurenyu** [comment]
closing in favor of #66 (sorry Yang...)

**Pull request closed without merging**